graylog2-server/src/test/java/org/graylog2/ElasticsearchBase.java

```diff
@@ -57,6 +60,8 @@
 
 public abstract class ElasticsearchBase {
     private static final String PROPERTIES_RESOURCE_NAME = "elasticsearch.properties";
+    private static final String DEFAULT_PORT = "9200";
```

Comment:
Shouldn't it be `19200` now?

Reply:
No, because if you run the tests from the IDE, you usually want to use the default Elasticsearch HTTP port.

When the tests run in context of the `integration-test` phase of Maven, the property with the port of the Elasticsearch HTTP API of the "test" instance will be there and override the deafult.

Reply:
:+1: